CommandHelper/LocalPackages/commands/buffs/optimus.ms

```diff
@@ -0,0 +1,24 @@
+@hero = associative_array(
```

Comment:
the top of this file is totally fucked

I suggest you look at a different hero and make it look like that